omero_figure/views.py

```diff
@@ -130,6 +130,10 @@ def img_data_json(request, image_id, conn=None, **kwargs):
         rv['pixel_size']['valueY'] = py.getValue()
         rv['pixel_size']['symbolY'] = py.getSymbol()
         rv['pixel_size']['unitY'] = str(py.getUnit())
+        pz = image.getPrimaryPixels().getPhysicalSizeZ()
+        rv['pixel_size']['valueZ'] = pz.getValue()
```

Comment:
You'll need to check `if pz is not None:` before using it, since for some 2D images they may have pixel sizes for X and Y, but not for Z.